apps/framework-cli/src/cli/routines/metrics_console/run_console/client.rs

```diff
@@ -21,6 +21,8 @@ pub struct ParsedMetricsData {
     pub histogram_vec: Vec<Sample>,
     pub kafka_messages_in_total: HashMap<String, (String, f64)>,
     pub kafka_messages_out_total: Vec<(String, String, f64)>,
+    pub flows_messages_in: HashMap<String, f64>,
```

Comment:
We probably should call those variables streaming_functions since that's they are now called in the documentation